package.json

```diff
@@ -6,15 +6,24 @@
     "autoprefix": "postcss --use autoprefixer -o network-api/networkapi/frontend/_css/main.compiled.css network-api/networkapi/frontend/_css/main.compiled.css",
     "build:common": "shx rm -rf network-api/networkapi/frontend && shx mkdir -p network-api/networkapi/frontend/_js && run-p build:images build:sass && npm run build:contribute",
     "build:contribute": "shx cp contribute.json network-api/networkapi/frontend/",
+    "build:dev": "run-s build:common build:js:dev",
+    "build:dev:a11y": "npm run cypress:install && npm run cypress:open",
     "build:images": "shx rm -rf network-api/networkapi/frontend/_images && shx cp -r source/images network-api/networkapi/frontend/_images",
-    "build:js-uncompressed": "webpack",
-    "build:js": "webpack -p",
+    "build:js-uncompressed": "npm run build:js:dev",
+    "build:js": "webpack --mode=production",
+    "build:js:dev": "webpack --mode=development",
     "build:sass": "shx mkdir -p network-api/networkapi/frontend/_css && node-sass source/sass/main.scss network-api/networkapi/frontend/_css/main.compiled.css && node-sass source/sass/buyers-guide/bg-main.scss network-api/networkapi/frontend/_css/buyers-guide.compiled.css && npm run autoprefix",
-    "build": "npm run build:common && npm run build:js",
-    "build-uncompressed": "npm run build:common && npm run build:js-uncompressed",
+    "build": "run-s build:common build:js",
+    "build-uncompressed": "run-s build:common build:js-uncompressed",
     "cypress": "run-p --race server cypress:test",
-    "cypress:install": "npm install --no-save cypress@^3.7.0 @percy/cypress@^2.0.0",
-    "cypress:test": "wait-on tcp:8000 && cypress run",
+    "cypress:install": "run-s cypress:install:*",
+    "cypress:install:cypress": "npm list cypress || npm install cypress@^3.7.0 --no-save",
+    "cypress:install:percy": "npm list @percy/cypress|| npm install @percy/cypress@^2.0.0 --no-save",
+    "cypress:install:cypress-axe": "npm list cypress-axe || npm install cypress-axe@^0.5.3 --no-save",
+    "cypress:install:testing": "npm list @testing-library/cypress || npm install @testing-library/cypress@^5.0.2 --no-save",
```

Comment:
This seems to run for 30s for each command, making `npm run build:dev:a11y` take over two minutes to launch the test browser. do we need these all to be separate commands?

Reply:
We can probably skip the `cypress:install` as part of the run, now that we have it as separate instruction.